recipes/gmx_MMPBSA/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "gmx_MMPBSA" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b166ec0a6a83d86e00865f54177dc99936882f4e2c478fedf35834eac0574f09
+
+build:
+  number: 0
+  skip: True  # [win]
+  noarch: python
```

Comment:
Why are you skipping Windows? If it is a missing dependency we can remove the skip and let conda solver deal with it to keep it noarch.

Reply:
Interestingly, windows checking failed unexpectedly on recipe generation, but then finished successfully

Reply:
I will try your suggestion